packages/core/src/functions.scss

```diff
@@ -0,0 +1,9 @@
+
+/**
+ * Function for converting a px size to rem;
+ */
+@function rem($size) {
```

Comment:
Maybe rename `$size` to `$pxSize`? A bit nitpicky, but since unitless values don't work any clarification is good (it shows up in intellisense) 😊

Reply:
Not nitpicky at all - I agree. Fixed.